src/providers/oauth-token-provider.js

```diff
@@ -101,10 +103,71 @@ function OAuthTokenProvider() {
        */
 
       removeToken() {
-        return ipCookie.remove(config.name, config.options);
+        return removeToken();
       }
+
     }
 
+    /**
+     * setToken
+     *
+     * @param data
+     * @returns {*}
+     */
+    var setToken = function(data) {
```

Comment:
I agree with @ruipenso, something like `oauthStorageService`, which would abstract the storage of the token.